funnel/views/helpers.py

```diff
@@ -20,3 +24,64 @@ def localize_date(date, from_tz=utc, to_tz=utc):
             to_tz = pytz_timezone(to_tz)
         return from_tz.localize(date).astimezone(to_tz).replace(tzinfo=None)
     return date
+
+def basepath(url):
+    """
+    Returns the base path of a given a URL
+    Eg::
+
+        basepath("https://hasgeek.com/1")
+        >> u"https://hasgeek.com
+
+    :param url: A valid URL unicode string. Eg: https://hasgeek.com
+    """
+    parsed_url = urlparse(url)
+    if not (parsed_url.scheme or parsed_url.netloc):
+        raise ValueError("Invalid URL")
+    return u"{scheme}://{netloc}".format(scheme=parsed_url.scheme, netloc=parsed_url.netloc)
+
+def cors(any_get=False):
```

Comment:
This decorator should be in `coaster.views` as it's a dupe of what Boxoffice has.